polkadb/database.go

```diff
@@ -18,76 +18,111 @@ package polkadb
 
 import (
 	"os"
+	"path/filepath"
 
 	log "github.com/ChainSafe/log15"
 	"github.com/dgraph-io/badger"
 	"github.com/golang/snappy"
 )
 
-// BadgerService contains directory path to data and db instance
-type BadgerService struct {
+// Db contains directory path to data and db instance
+type Db struct {
```

Comment:
The ordering of this file is a bit confusing. Please try grouping types with their methods. Might be helpful to move some things to separate file